samples/meetings-app-icon-badging/nodejs/bots/teamsBot.js

```diff
@@ -0,0 +1,127 @@
+                    await context.sendActivity({ attachments: [this.createMembersAdaptiveCard(members)] });
+                }
+                else {
+                    await context.sendActivity("Please type `SendNotification` to send In-meeting notifications.");
+                }
+            }
+            else if (context.activity.value.Type == "StageViewNotification") {
+                var adaptiveCardChoiceSet = context.activity.value.Choice;
+                var selectedMembers = adaptiveCardChoiceSet.split(",");
+                this.stageView(context, meetingId, selectedMembers);
+            }
+            else if (context.activity.value.Type == "AppIconBadging") {
+                var adaptiveCardChoiceSet = context.activity.value.Choice;
+                var selectedMembers = adaptiveCardChoiceSet.split(",");
+                this.visualIndicator(context, meetingId, selectedMembers);
+            }
+
+            await next();
+        });
+    };
+
+    // Custom method for sending targeted meeting notifications in stage.
+    async stageView(context, meetingId, selectedMembers) {
+
+        // Notification payload for meeting target notification API.
+        let notificationInformation = {
+            type: "targetedMeetingNotification",
+            value: {
+                recipients: selectedMembers,
+                surfaces: [
+                    {
+                        surface: "meetingStage",
+                        contentType: "task",
+                        content: {
+                            value: {
+                                height: "300",
+                                width: "400",
+                                title: "Targeted meeting Notification",
+                                url: `${this.baseUrl + "/hello.html"}`
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+
+        try {
+            await TeamsInfo.sendMeetingNotification(context, notificationInformation, meetingId);
+        } catch (exception) {
+            console.log(exception);
+        }
+    }
+
+    // Custom method for sending App Icon Badging on tab.
+    async visualIndicator(context, meetingId, selectedMembers) {
+
+        // Notification payload for meeting target notification API.
+        let notificationInformation = {
+            type: "targetedMeetingNotification",
+            value: {
+                recipients: selectedMembers,
+                surfaces: [
+                    {
+                        surface: "meetingTabIcon",
+                    }
+                ]
+            }
+        }
+
+        try {
+            await TeamsInfo.sendMeetingNotification(context, notificationInformation, meetingId);
+        } catch (exception) {
+            console.log(exception);
+        }
+    }
+
+    // Create adaptive card for send list of in-meeting participants.
```

Comment:
nit typo - Create an adaptive card to send a list of in-meeting participants

Reply:
updated